spec/javascripts/govuk-component/option-select-spec.js

```diff
@@ -86,6 +86,17 @@ describe('GOVUK.OptionSelect', function() {
     expect(optionSelect.isClosed()).toBe(false);
   });
 
+  it ('sets the height of the options container as part of initialisation', function(){
```

Comment:
These tests could communicate the bug we're fixing, if we named this test against how we're checking the component is expanded correctly we'd notice that we're testing the height is set but we're not testing that the aria-expanded attribute is set properly.

Reply:
We need a separate second test to check that aria-expanded has the correct value(s)

Reply:
The first two tests in the file now cover this for both open and closed initial states.